### PR TITLE
Catch exceptions when running a deployment config

### PR DESCRIPTION
**Pull request by etanshaul**
This is not my ideal solution, but it does suppress the IDE error when a user attempts to run with an invalid deployment config. 

I attempted to fix this at the source - force validation in the dialog when opening with a programatically created configuration to no avail. The bundled app engine plugin also suffers from this problem (and doesn't even suppress the IDE error). I think this the minimum we should do - suggestions welcome.

### CODE REVIEW

**Comment by etanshaul** [comment]
fixes #635

File: google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineToolsMenuAction.java [hunk 1]
```diff
@@ -52,8 +55,12 @@ public void actionPerformed(AnActionEvent e) {
       List<RemoteServer<AppEngineServerConfiguration>> servers =
           RemoteServersManager.getInstance().getServers(serverType);
 
-      DeploymentConfigurationManager.getInstance(project).
-          createAndRunConfiguration(serverType, ContainerUtil.getFirstItem(servers));
+      try {
+        DeploymentConfigurationManager.getInstance(project).
+            createAndRunConfiguration(serverType, ContainerUtil.getFirstItem(servers));
+      } catch (Exception ex) {
```
Comment (patflynn) [inline google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineToolsMenuAction.java]: Can you catch NPE and then in the catch block include a comment pointer to the issue?

**Comment by etanshaul** [comment]
good suggestion. done.

**Comment by patflynn** [comment]
LGTM

**Pull request merged**